pdg/src/main.rs

```diff
@@ -107,7 +112,7 @@ fn main() -> eyre::Result<()> {
             .map(|node_id| node_id.as_usize())
             .collect::<Vec<_>>();
         if should_print(ToPrint::Graphs) {
-            println!("{graph}");
+            println!("Graph:\n {graph}");
```

Comment:
```suggestion
            println!("{graph}");
```

Do we need this?  The `g {` printed is meant to mean it's a `Graph`, like how `n[id]` is for a `NodeId`.  Originally we'd print the `GraphId` similarly, too, but that caused unnecessarily noisy diffs and we didn't really need it so we removed that.